### DISCUSSION TITLE
Copilot in VS Code - reading suggestions to a screen reader

### DISCUSSION THREAD

**Discussion by benjiallen** [community/community#17974] [Accessibility]
Quoting @jeanbrazcosta from [comment on June 4 2022](https://github.com/orgs/github-community/discussions/16996#discussioncomment-2885004)

> Good morning, I'm blind, and I've been using copillot in vscode, along with the NVDA screen reader.
> 
> It happens that, when opening a java class and starting to type, I hear a sound from the extension informing that there is a suggestion, but I don't have a shortcut or command for the reader to read the suggestion.
> 
> I can only see what it is, when I press tab or when I press high + enter.
> 
> If anyone can help, I appreciate it!

**Comment by benjiallen** [comment]
Hi @jeanbrazcosta, thanks for the feedback. Let me reach out to some colleagues to see if I can find some help for your query.

**Reply by jeanbrazcosta** [reply]
Thanks for the feedback @benjiallen, I reported this same issue to the VsCode team, but they told me I should open this improvement request directly to the copilot extension team.

Reading the documentation, I saw that it has the shortcut control + [ so that it displays a new suggestion when accepting with the tab, but even using this shortcut the screen reader didn't read the suggestion before I inserted it into the class.

If you need any more information, I'm at your disposal.

**Comment by cehfisher** [comment]
Hello @jeanbrazcosta - I wanted to check in on this issue. Have the recent changes to GitHub Copilot resolved it for you? If so, could you please close this issue? If not, do you have any updates to add to what you already reported? Thank you for your feedback! (cc @queenofcorgis)

**Comment by jeanbrazcosta** [comment]
Yes, I will close this issue , as the updates have already resolved what I reported here

**Comment by jeanbrazcosta** [comment]
Can anyone close this topic please?

I don't have the option to do that.

**Reply by queenofcorgis** [reply]
I got you @jeanbrazcosta - only the original poster or a member of the Community team can close posts 😃
